types/picomatch/lib/picomatch.d.ts

```diff
@@ -31,7 +31,7 @@ declare namespace picomatch {
     type Glob = string | string[];
 
     interface Matcher {
-        (test: string): boolean;
+        (test: string, returnObject?: boolean): boolean;
```

Comment:
The return type changes when `returnObject` is true: https://github.com/micromatch/picomatch/blob/master/lib/picomatch.js#L75

We probably need to create a new interface which is the return type if `returnObject` is true. Based on this: https://github.com/micromatch/picomatch/blob/master/lib/picomatch.js#L67